pkg/controller/kubelet-config/helpers.go

```diff
@@ -23,6 +23,29 @@ import (
 	mcfgclientset "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned"
 )
 
+func createNewKubeletDynamicSystemReservedIgnition(dynamicSystemReserved bool) *ign3types.File {
+	config := fmt.Sprintf("NODE_SIZING_ENABLED=%s\n", strconv.FormatBool(dynamicSystemReserved))
+
+	mode := 0644
+	overwrite := true
+	du := dataurl.New([]byte(config), "text/plain")
+	du.Encoding = dataurl.EncodingASCII
+	duStr := du.String()
+
+	return &ign3types.File{
+		Node: ign3types.Node{
+			Path:      "/etc/node-sizing-enabled.env",
```

Comment:
this should be in /etc/kubernetes

Reply:
When the file `node-sizing-enabled.env` is moved in /etc/kubernetes, it gets `system_u:object_r:kubernetes_file_t:s0` SELinux labels. For some reason, the system unit file isn't able to import this file,

```
# systemctl status kubelet-auto-node-size.service
● kubelet-auto-node-size.service - Dynamically sets the system reserved for the kubelet
   Loaded: loaded (/etc/systemd/system/kubelet-auto-node-size.service; enabled; vendor preset: disabled)
   Active: failed (Result: resources)
      CPU: 0

Mar 22 11:35:11 testclustllb1-hltqw-worker-a-klvm6 systemd[1]: kubelet-auto-node-size.service: Failed to load environment files: Permission denied
Mar 22 11:35:11 testclustllb1-hltqw-worker-a-klvm6 systemd[1]: kubelet-auto-node-size.service: Failed to run 'start' task: Permission denied
Mar 22 11:35:11 testclustllb1-hltqw-worker-a-klvm6 systemd[1]: kubelet-auto-node-size.service: Failed with result 'resources'.
Mar 22 11:35:11 testclustllb1-hltqw-worker-a-klvm6 systemd[1]: Failed to start Dynamically sets the system reserved for the kubelet.
Mar 22 11:41:38 testclustllb1-hltqw-worker-a-klvm6 systemd[1]: kubelet-auto-node-size.service: Failed to load environment files: Permission denied
Mar 22 11:41:38 testclustllb1-hltqw-worker-a-klvm6 systemd[1]: kubelet-auto-node-size.service: Failed to run 'start' task: Permission denied
Mar 22 11:41:38 testclustllb1-hltqw-worker-a-klvm6 systemd[1]: kubelet-auto-node-size.service: Failed with result 'resources'.
Mar 22 11:41:38 testclustllb1-hltqw-worker-a-klvm6 systemd[1]: Failed to start Dynamically sets the system reserved for the kubelet.
```

Looking into it.

Reply:
`aureport` confirms it,

```
12. 03/22/2021 11:35:11 systemd system_u:system_r:init_t:s0 0 file read system_u:object_r:kubernetes_file_t:s0 denied 45
13. 03/22/2021 11:41:38 systemd system_u:system_r:init_t:s0 0 file read system_u:object_r:kubernetes_file_t:s0 denied 86
```

Reply:
That env file is created on first boot using, https://github.com/openshift/machine-config-operator/pull/2466/files#diff-c56bef4bc0ff73a538d970d1b4d9684b3272703584bbfe8002dc9f5c81368adfR1

Reply:
I haven't updated the changes that move that env file from `/etc/` to `/etc/kubernetes` in github. The failures that I have pasted above are from my local setup after making the changes. So in this PR you are still seeing `/etc/` paths and not `/etc/kubernetes`